opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -16,8 +16,11 @@
 
 from wrapt import ObjectProxy
 
+from opentelemetry.context import create_key
 from opentelemetry.trace import StatusCode
 
+_SUPPRESS_INSTRUMENTATION_KEY = create_key("suppress_instrumentation")
```

Comment:
Can we leave a comment saying that this is a temporary solution, and we are waiting on a decision to how we are handling supressing instrumentations?